static/js/components/SkyCam.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import CardMedia from "@material-ui/core/CardMedia";
+
+const useStyles = makeStyles(() => ({
+  cardDiv: {
+    minWidth: "18.75rem",
+    minHeight: "18.75rem",
+  },
+  title: {
+    fontSize: "0.875rem",
+  },
+  media: {
+    minHeight: "18.75rem",
+  },
+}));
+
+const SkyCam = ({ telescope }) => {
+  const classes = useStyles();
+
+  if (!telescope.skycam_link) {
+    return <div />;
+  }
+
+  const handleImageError = (e) => {
+    e.target.onerror = null;
+    e.target.src =
+      "https://ak.picdn.net/shutterstock/videos/5030309/thumb/1.jpg";
```

Comment:
Should not be hardcoding in URLs without a guarantee that they will be there. Suggest downloading and storing this Not Found image locally in the repo. Might as well use an animated gif, right? (https://www.google.com/search?q=static+TV+animated+gif)

Reply:
Animated gif will significantly increase the size of the git repo (by ~0.5MB), so keeping to a static image in `static/images/`.